docs/gcloud-auth.rst

```diff
@@ -0,0 +1,208 @@
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+Authentication
+--------------
+
+=======
+Summary
+=======
+
+For the majority of cases,
```

Comment:
How about we make it clear "If you're running in GAE, or GCE or authenticated with the Cloud SDK, [authentication should 'just work']" ?

It should always work -- in the cases below we describe them -- but we want to say that in these situations you don't even have to think about it.